components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -4365,6 +4467,87 @@ private int getAuthentictorID(Connection conn, int tenantId, String idpName,
         return authId;
     }
 
+    /**
+     * Retrieves the IDP ID for the given IDP name.
+     *
+     * @param conn Database connection.
+     * @param tenantId Tenant Id of the application.
+     * @param idpName Identity Provider Name.
+     * @return Identity Provider ID.
+     * @throws SQLException SQL Exception if error occurred while retrieving IDP ID.
+     */
```

Comment:
```suggestion
    /**
     * Retrieve the IDP ID for the given IDP name.
     *
     * @param conn     Database connection.
     * @param tenantId Tenant Id of the application.
     * @param idpName  Identity Provider Name.
     * @return Identity Provider ID.
     * @throws SQLException SQL Exception if error occurred while retrieving IDP ID.
     */
```

Reply:
Addressed in 4cdf9672bdd6a4d6060995024d14bd9dc79a1168